mkdocs-base.yml

```diff
@@ -147,7 +147,7 @@ nav:
   - Welcome to Percona Everest: index.md 
   - Release Notes:
       - Release notes index: release-notes/release_notes_index.md
-      - Percona-Everest 1.1.0 (2024-07-26): release-notes/Percona-Everest-1.1.0-(2024-07-26).md
```

Comment:
Release date has now been moved to monday, aug 12th

Reply:
Changed the date.